src/ir/value.h

```diff
@@ -115,13 +139,22 @@ class Value {
   }
 
   // A downcast operation. Just delegates directly to std::get on variants.
-  template<class T>
-  const T &As() const { return std::get<T>(value_); }
+  template <class T>
```

Comment:
Didn't we decide to get rid of this method?

Reply:
Maybe. It does seem that `If` is proving to be more useful. I can do that in a separate PR; the changes here only occurred due to formatting.